lib/matplotlib/figure.py

```diff
@@ -1679,6 +1679,9 @@ def subplot_mosaic(self, mosaic, *, sharex=False, sharey=False,
            This API is provisional and may be revised in the future based on
            early user feedback.
 
+        See :doc:`/tutorials/provisional/mosaic`
+        for an example of its usage and provisional API documentation
+
```

Comment:
I would just say "... for an example."  Or if you really want.  ... for an example and full API explanation"

Reply:
Done :)